### PR TITLE
Environmentvariables

### PR DESCRIPTION
**Pull request by RajRele**
Changes related to the efforts to save the Demo Test Account integration information in a env variables encrypted in the AppVeyor CI

### CODE REVIEW

File: test/SdkNetCoreTests/TestConfig.cs [hunk 1]
```diff
@@ -20,46 +21,52 @@ class TestConfig
...
         {
+            string userIdFromEnv = Environment.GetEnvironmentVariable("userid");
+            string integratorKeyFromEnv = Environment.GetEnvironmentVariable("integratorkey");
+            string templateIdFromEnv = Environment.GetEnvironmentVariable("templateid");
+            string integratorKeyNoConsentFromEnv = Environment.GetEnvironmentVariable("integratorkeynoconsent");
+            string brandIdfromEnv = Environment.GetEnvironmentVariable("brandid");
+
             this.Host = (host != null) ? host : "https://demo.docusign.net/restapi";
-            this.Username = (username != null) ? username : "node_sdk@mailinator.com";
-            this.Password = (password != null) ? password : "{PASSWORD}";
```
Comment (gsnavin) [inline test/SdkNetCoreTests/TestConfig.cs]: Remove all references to password. We don't need it anymore.

File: test/SdkTests/TestConfig.cs [hunk 2]
```diff
@@ -20,46 +21,52 @@ class TestConfig
...
         {
+            string userIdFromEnv = Environment.GetEnvironmentVariable("userid");
+            string integratorKeyFromEnv = Environment.GetEnvironmentVariable("integratorkey");
+            string templateIdFromEnv = Environment.GetEnvironmentVariable("templateid");
+            string integratorKeyNoConsentFromEnv = Environment.GetEnvironmentVariable("integratorkeynoconsent");
+            string brandIdfromEnv = Environment.GetEnvironmentVariable("brandid");
+
             this.Host = (host != null) ? host : "https://demo.docusign.net/restapi";
-            this.Username = (username != null) ? username : "node_sdk@mailinator.com";
-            this.Password = (password != null) ? password : "{PASSWORD}";
```
Comment (gsnavin) [inline test/SdkTests/TestConfig.cs]: Same here

File: test/SdkNetCoreTests/TestConfig.cs (hunk 1 above)
Comment (RajRele) [reply test/SdkNetCoreTests/TestConfig.cs]: removed!

File: test/SdkTests/TestConfig.cs (hunk 2 above)
Comment (RajRele) [reply test/SdkTests/TestConfig.cs]: removed!

File: test/SdkNetCoreTests/TestConfig.cs [hunk 3]
```diff
@@ -20,46 +21,50 @@ class TestConfig
...
         public string IntegratorKeyNoConsent { get; set; }
 
-        public string PrivateKeyNoConsentFilename { get; set; }
+        public string PrivateKeyNoConsent { get; set; }
 
         public string BrandLogo { get; set; }
 
         public string BrandId { get; set; }
 
         public TestConfig(string username = null, string password = null, string integratorKey = null, string host = null, string recipientEmail = null, string recipientName = null, string templateRoleName = null, string templateId = null, string returnUrl = null)
```
Comment (gsnavin) [inline test/SdkNetCoreTests/TestConfig.cs]: Its still here :(

File: test/SdkTests/TestConfig.cs [hunk 4]
```diff
@@ -20,46 +21,50 @@ class TestConfig
...
         public string IntegratorKeyNoConsent { get; set; }
 
-        public string PrivateKeyNoConsentFilename { get; set; }
+        public string PrivateKeyNoConsent { get; set; }
 
         public string BrandLogo { get; set; }
 
         public string BrandId { get; set; }
 
         public TestConfig(string username = null, string password = null, string integratorKey = null, string host = null, string recipientEmail = null, string recipientName = null, string templateRoleName = null, string templateId = null, string returnUrl = null)
```
Comment (gsnavin) [inline test/SdkTests/TestConfig.cs]: Same here!

Comment (RajRele) [reply test/SdkTests/TestConfig.cs]: done!

File: test/SdkNetCoreTests/TestConfig.cs (hunk 3 above)
Comment (RajRele) [reply test/SdkNetCoreTests/TestConfig.cs]: removed!

**Pull request merged**